index.html

```diff
@@ -5,330 +5,360 @@
-                                    <h4>Back End Dev</h4>
-                                </div>
-                            </div>
-                            <div class="counterimage1"><img src="./Images/Counter.png" alt=""></div>
-                            <div class="text3">
-                                <h4>CANOPY</h4>
-                            </div>
+                                </li>
+                                <li class="counter"> <img src="./Images/Counter.png" alt="counter"></li>
+                                <li class="roletext">
+                                    <h4>Lead Developer</h4>
+                                </li>
+                                <li class="counter"><img src="./Images/Counter.png" alt="counter"></li>
+                                <li class="canopy">
+                                    <h5>Uber</h5>
+                                </li>
+                            </ul>
                         </div>
-                        <div class="project3">
-                            <h3>Multi-Post Stories</h3>
+                    </div>
+                    <div class="daily">
+                        <div class="midtext">
+                            <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
                         </div>
-
                     </div>
-                    <div class="primetxt3">
-                        <p> A daily selection of privately personalized <br> reads; no accounts or <br> sign-ups required.</p>
+                    <div class="action1"></div>
+                    <div class="tags">
+                        <ul>
+                            <li>
+                                <p>html</p>
+                            </li>
+                            <li>
+                                <p>css</p>
+                            </li>
+                            <li>
+                                <p>javascript</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="nbtn">
+                        <button class="see">See Project</button>
                     </div>
                 </div>
-            </div>
-            <div class="container4">
-                <div id="atext" class="atexts">
-                    <h2>About Myself</h2>
-                </div>
-                <div class="about">
-                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="ptext">
                 </div>
+                <div class="containerbasic"></div>
+            </div>
+        </section>
+        <section class="about">
+            <div class="aboutcontainer">
```

Comment:
- [ ] On the desktop screen, there are 2 elements, the left one(about myself) should take 40% of the available width and the right one(languages, skills, frameworks) the remaining 60% of the page's width, and the contents of each of them should be aligned to the right.
![image](https://user-images.githubusercontent.com/33767951/140236109-19ec94f3-cb7a-40de-bf9d-fa5a96b9b803.png)